app/screens/recipes/RecipesMainScreen.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  Text, StyleSheet, View, Button, TouchableOpacity,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  text: {
+    fontSize: 30,
+  },
+  touchable: {
+    width: 300,
+    marginTop: 556,
+    marginLeft: 25,
+    paddingVertical: 18,
+    borderWidth: 2,
+    borderColor: '#20232a',
+    borderRadius: 6,
+    backgroundColor: '#61dafb',
```

Comment:
You should add the colors to the constants/colors.js file to ensure everyone is using the same color blue.
https://www.color-name.com/hex/61dafb
<img width="572" alt="Screenshot 2023-06-11 at 1 13 59 PM" src="https://github.com/Full-Metal-Alchemist/NativEats/assets/6126925/1781ddcd-c88c-4d8a-843b-0d3e1a805124">

```
import {COLORS} from '../../constants/colors';
... 
backgroundColor: COLORS.SKY_BLUE,